packages/base/command-result.gts

```diff
@@ -129,17 +130,20 @@ class CommandResultEmbeddedView extends Component<typeof CommandResult> {
               {{cardResource.cardError.error.message}}
             </div>
           {{else if cardResource.card}}
-            <div
-              class='card-item'
-              data-test-result-card={{cardResource.card.id}}
-              data-test-result-card-idx={{i}}
-            >
-              {{#let (getComponent cardResource.card) as |Component|}}
-                {{i}}.
-                <Component @format='atom' @displayContainer={{false}} />
-              {{/let}}
-            </div>
+            {{#let (add i 1) as |idx|}}
+              <div
+                class='card-item'
+                data-test-result-card={{cardResource.card.id}}
+                data-test-result-card-idx={{idx}}
+              >
+                {{#let (getComponent cardResource.card) as |Component|}}
+                  {{idx}}.
```

Comment:
This works, though I don’t see any test assertions changing though for the `data-test-result-card-idx=`, could we use `<ol>` instead?

Reply:
Thanks. I did this and removed idx all together